app/src/main/kotlin/de/digitalService/useID/ui/dialogs/ScanErrorDialog.kt

```diff
@@ -68,8 +69,11 @@ fun ScanErrorAlertDialog(error: ScanError, onButtonTap: () -> Unit) {
                     Spacer(modifier = Modifier.height(16.dp))
                 }
 
+                val packageName = LocalContext.current.packageName
+                val imagePath = "android.resource://$packageName/${R.drawable.nfc_positions}"
+
                 MarkdownText(
-                    markdown = markDownResource(id = error.textResID),
+                    markdown = markDownResource(id = error.textResID, imagePath),
```

Comment:
Would be nice to also use the `markDownResource` in the ScanScreen and maybe have a way to pass images to it, so that we can move the packageName and imagePath code to that function.

Reply:
Definitely. Will be part of the refactoring that will overhaul the whole UI handling.